MH-FiorellaTheme5/source/partials/_top-navigation.erb

```diff
@@ -0,0 +1,30 @@
+<div class="topnav">
+  <div class="burger-menu">
+    <i class="fas fa-bars" onclick="openNav()"></i>
```

Comment:
Avoid calling functions inside HTML, instead of that set a class or id to this element and set a listener (`click`), and inside that listener call the `openNav()` function